cmd/client/packet_injector.go

```diff
@@ -203,11 +272,31 @@ func addInjectPacketFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint8VarP(&ttl, "ttl", "", 64, "time-to-live")
 }
 
+func addInjectPacketInjectFlags(cmd *cobra.Command) {
```

Comment:
create a helper function to contain duplicated code for this func and the func above